### PR TITLE
Allow manual port mapping when using "run" command

### PR DESCRIPTION
**Pull request by mrfuxi**
Support mapping specific ports in "docker-compose run" #1709 

Signed-off-by: Karol Duleba <mr.fuxi@gmail.com>

### CODE REVIEW

**Comment by thaJeztah** [comment]
Can you edit the description above, and add `fixes #1709`? That will make sure that the issue will automatically closed when your PR is merged :+1:

**Comment by mrfuxi** [comment]
Work is ready for review

File: docs/reference/run.md [hunk 1]
```diff
@@ -38,6 +39,10 @@ The second difference is the `docker-compose run` command does not create any of
 
     $ docker-compose run --service-ports web python manage.py shell
 
+Alternatively manual port mapping can be specified. Same as when running Docker's `run` command - using `--publish` or `-p` options:
+
+    $ docker-compose run --publish 8080:80 -p 2022:22 web python manage.py shell
```
Comment (thaJeztah) [inline docs/reference/run.md]: Should this also support the `ip:hostPort:containerPort` form (i.e. allow specifying an IP-address)? (t.b.h., I never needed it, but others may)

Comment (aanand) [inline docs/reference/run.md]: From the code it looks like it will - we should document and test it.

**Comment by mrfuxi** [comment]
test/docs for `ip:hostPort:containerPort` added

**Comment by aanand** [comment]
Thanks!

ping @moxiegirl - docs look OK?

**Comment by moxiegirl** [comment]
Thanks @mrfuxi LGTM

**Comment by aanand** [comment]
LGTM

**Comment by dnephin** [comment]
LGTM

**Comment by bfirsh** [comment]
yessss :+1:

**Comment by bfirsh** [comment]
Thanks @mrfuxi! I've wanted this for ages.

**Comment by thaJeztah** [comment]
Thanks so much! :tada:

**Pull request merged**